modules/tasks/tasks/src/main/resources/webapp/js/directives.js

```diff
@@ -794,6 +794,51 @@
         };
     });
 
+
+    directives.directive('datetimePickerStartDateInput', function () {
```

Comment:
Do we really need to keep multiplying these datepicker directives? No way to reuse the existing ones?